src/plugins/gc-subway/_base.scss

```diff
@@ -16,46 +42,47 @@
 		border-top: 4px solid #26374a;
```

Comment:
The border should not be visible on mobile mode. See image that shows the border:

<img width="743" alt="Screen Shot 2020-12-06 at 7 06 57 PM" src="https://user-images.githubusercontent.com/4353199/101297803-d16e0e80-37f8-11eb-934b-056149de66b4.png">

Reply:
I am looking at it